.circleci/config.yml

```diff
@@ -1191,7 +1191,7 @@ workflows:
   nightly:
     triggers:
       - schedule:
-          cron: "* 18 * * *"
+          cron: "18 11 * * *"
```

Comment:
Should these be flipped for "nightly"? https://www.unix.com/man-page/POSIX/1posix/crontab/

Reply:
This is early morning in UTC. So I want result ready every morning.

Reply:
I forgot it's UTC :)